server/planning/content_api/events/event.py

```diff
@@ -21,7 +21,15 @@
 
 
 class ContentAPIEventService(AsyncResourceService[ContentAPIEventResource]):
-    pass
+    async def publish_async(self, item, subscribers=None) -> None:
```

Comment:
Won't the item passed in would still be in the original internal Event & Planning format?
I believe we may have to format the item based on [json_event](https://github.com/superdesk/superdesk-planning/blob/async/server/planning/output_formatters/json_event.py) and [json_planning](https://github.com/superdesk/superdesk-planning/blob/async/server/planning/output_formatters/json_planning.py) to convert the item into the correct ContentAPI format, similar to what we do with [content formatting](https://github.com/superdesk/superdesk-core/blob/async/content_api/publish/service.py#L81)

Reply:
yeah, they are but in the end both content_api planning and event models inherit from both "internal" event & planning models therefore, they share all the same fields so I was not sure which fields should be preserved and which aren't.

Reply:
@MarkLark86 I've implemented the changes to format the items before creating/updating them. Please take another look.